docs/AwsMarketplaceGuide.md

```diff
@@ -0,0 +1,207 @@
+
+1. Update the custom values file of the Helm Chart of a Scalar product you want to install.  
+   You need to specify the private container registry (ECR) of AWS Marketplace and the version (tag) as the value of `[].image.repository` and `[].image.version (tag)` in the custom values file.  
+   * Examples
+       * Scalar DB
+           * Scalar DB Server (scalardb-custom-values.yaml)
+             ```yaml
+             envoy:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardb-envoy"
+                 version: "1.2.0"
+             
+             ...
+             
+             scalardb:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardb-server"
+                 tag: "3.5.2"
+             ```
+       * Scalar DL
+           * Scalar DL Ledger (scalardl-ledger-custom-values.yaml)
+             ```yaml
+             envoy:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-ledger-envoy"
+                 version: "1.2.0"
+             
+             ...
+             
+             ledger:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-ledger"
+                 version: "3.4.1"
+             ```
+           * Scalar DL Auditor (scalardl-auditor-custom-values.yaml)
+             ```yaml
+             envoy:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor-envoy"
+                 version: "1.2.0"
+             
+             ...
+             
+             auditor:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor"
+                 version: "3.4.1"
+             ```
+           * Scalar DL Schema Loader for Ledger (schema-loader-ledger-custom-values.yaml)
+             ```yaml
+             schemaLoading:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardl-schema-loader-ledger"
+                 version: "3.4.1"
+             ```
+           * Scalar DL Schema Loader for Auditor (schema-loader-auditor-custom-values.yaml)
+             ```yaml
+             schemaLoading:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardl-schema-loader-auditor"
+                 version: "3.4.1"
+             ```
+
+1. Deploy the Scalar product using the Helm Chart with the above custom values file.
+   * Examples
```

Comment:
Let's use the same structure as above for clarity.
(The above custom values files are kind of grouped into Scalar DB and Scalar DL, so let's make this also look the same)

Reply:
I updated it in 22d1432c1d0b1c987d47c4aa477e495982b749ed.